### PR TITLE
Revert "Sean/revert blog pages (#1869)"

### PR DESCRIPTION
**Pull request by sean-binary**
This reverts commit dfafd1d5f729d6ed3027ddd91fcfb3aba21edbb0.

Changes:

-   Add the articles and videos pages back onto the blog branch

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/besquare/components/sections/_what-is_besquare.js [hunk 1]
```diff
@@ -38,10 +38,15 @@ const WhatIsBeSquare = () => {
         <Section>
             <ContentContainer>
                 <ContentWrapper>
-                    <Title as="h2" text_align={'left'} max_width={['450px']} margin={['0', '0 auto']}>
+                    <Title
+                        as="h2"
+                        text_align={'left'}
+                        max_width={['450px']}
+                        margin={['0', '0 auto']}
```
Comment (mustofa-binary) [inline src/pages/besquare/components/sections/_what-is_besquare.js:45]: can it be an object?

File: src/pages/blog/articles/_featured-article.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                            </StyledCategories>
+                        )}
+                        <Header
+                            color="grey-5"
+                            as="h5"
+                            weight="normal"
+                            type="paragraph-2"
+                            width="auto"
+                        >
+                            {`• ${article_data[0].reading_time} min read`}
```
Comment (mustofa-binary) [inline src/pages/blog/articles/_featured-article.js:91]: hmm we might consider reading time as a string, (imagine translation for this)

File: src/pages/blog/videos/_video-card.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                    <PlayButtonOval />
+                    <StyledTriangle src={Triangle} alt="play button" height={25} width={20} />
+                    <VideoDuration as="h5" type="paragraph-2" weight="bold">
+                        {item.video_duration}
+                    </VideoDuration>
+                    <img
+                        src={item.image}
+                        alt="Video card"
+                        width="100%"
+                        style={{ objectFit: 'contain' }}
```
Comment (mustofa-binary) [inline src/pages/blog/videos/_video-card.js:140]: seems duplicated everywhere. we can make a styled component under blog/_components.js, and put Img with this style

File: src/pages/blog/videos/index.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        <SEO
+            title={localize('Latest videos, tutorials, webinars for trading | Deriv')}
+            description={localize(
+                "Learn how to trade using our trading videos and tutorials at Deriv's online trading academy.",
+            )}
+        />
+        <Hero jc="center" ai="center">
+            <SmallContainer>
+                <Header as="h2" type="heading-3" color="white" weight="400" align="left">
+                    Video tutorials
```
Comment (mustofa-binary) [inline src/pages/blog/videos/index.js:62]: if they want bilingual, we might need to localize this

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/pages/besquare/components/sections/_what-is_besquare.js (hunk 1 above)
Comment (sean-binary) [reply src/pages/besquare/components/sections/_what-is_besquare.js:45]: I didn't change this, just auto formatted. 😂 Not part of the scope.

File: src/pages/blog/articles/_data.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
+export const article_data = [
```
[the remark this answers is not in the document]
Comment (sean-binary) [reply src/pages/blog/articles/_data.js:1]: Yep it is, will be replaced with CMS data once ready.

File: src/pages/blog/videos/index.js (hunk 4 above)
Comment (sean-binary) [reply src/pages/blog/videos/index.js:62]: We'll handle this at a later phase right? You're saying we should just wrap with Localize now anyways?

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**